src/docregistry/plugin.ts

```diff
@@ -21,6 +21,16 @@ const docRegistryProvider: JupyterLabPlugin<IDocumentRegistry> = {
     let registry = new DocumentRegistry();
     registry.addModelFactory(new TextModelFactory());
     registry.addModelFactory(new Base64ModelFactory());
+    registry.addFileType({
+      name: 'Text',
+      extension: '.txt',
+      fileType: 'file',
+      fileFormat: 'text'
+    });
+    registry.addCreator({
+      name: 'Text File',
+      fileType: 'Text',
```

Comment:
@sccolbert - definitely confusing names here. Here is where the creator `fileType` is used to get the registered file type: https://github.com/blink1073/jupyterlab/blob/7b3eb36631ea0e2453fc11a3dcfac9986d81230f/src/filebrowser/dialogs.ts#L307

We should change one or the other names...